Examples/ExamplesApp/Examples.swift

```diff
@@ -37,20 +37,19 @@ struct Examples: View {
                         .navigationBarTitleDisplayMode(.inline)
                     }
                     .isDetailLink(false)
+                    .tag(category)
                 }
                 .navigationTitle("Toolkit Examples")
             }
         } detail: {
-            NavigationStack {
```

Comment:
I couldn't find a need for this `NavigationStack`. I tested on all supported platforms and didn't notice any broken behavior.